apps/mobile-wallet/src/components/text/LinkToWeb.tsx

```diff
@@ -16,18 +16,19 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+import { ReactNode } from 'react'
 import { Linking, StyleProp, TextStyle } from 'react-native'
 import styled from 'styled-components/native'
 
 import AppText from '~/components/AppText'
 
 interface LinkToWebProps {
-  text: string
   url: string
+  children?: ReactNode
```

Comment:
Shouldn't the children still be only `string`? Do we want to allow other components inside a `LinkToWeb` instance?

Reply:
I thought that it could indeed be handy, to wrap tags, buttons etc. with LinkToWeb.

Reply:
I think it's best to allow `ReactNode` when we actually need them for a more type-strict code-base. I change this discussion to `nit`. I'd do it if it was up to me, but it's up to you 😉